cmake/FindOpenEXR.cmake

```diff
@@ -198,6 +173,34 @@ if(EXISTS "${OpenEXR_INCLUDE_DIR}/OpenEXRConfig.h")
   set(OpenEXR_VERSION ${OpenEXR_VERSION_MAJOR}.${OpenEXR_VERSION_MINOR})
 endif()
 
+if(${OpenEXR_VERSION_MAJOR} MATCHES "3")
```

Comment:
Just as an aside, this is equivalent and perhaps more clear (or at least, more general, if you needed a divider at something other than a major version only):

    if (OpenEXR_VERSION VERSION_GREATER_EQUAL 3.0)

Reply:
I changed it to use VERSION_GREATER_EQUAL.